### PR TITLE
Axes Ratvar_Act() from the code

### PR DESCRIPTION
**Pull request by Sirryan2002**
## What Does This PR Do
removes `ratvar_act()` from multiple files

this DOES NOT remove brass from the game and still allows the construction of brass buildings. Dead god ruin(ratvar) is still there and still works.

## Why It's Good For The Game
Clockwork cult is never getting added to this codebase. It's just bloat code and brass rarely ever gets used except in events or when a miner occasionally takes the brass from the dead god ruin. However I recognize the fun/aesthetic of brass objects and left most of them in.

## Changelog
:cl:
del: strips ratvars powers even further
/:cl:

### CODE REVIEW

**Comment by hal9000PR** [comment]
`del: ratvar_act() from codebase` don't need this in the CL because it isn't player focused

**Comment by Sirryan2002** [comment]
> `del: ratvar_act() from codebase` don't need this in the CL because it isn't player focused

done

**Comment by AffectedArc07** [comment]
```
loading ./map_files/RandomRuins/LavaRuins/lavaland_surface_dead_ratvar.dmm
_maps/map_files/RandomRuins/LavaRuins/lavaland_surface_dead_ratvar.dmm:39:error: undefined type: /obj/structure/grille/ratvar/broken
_maps/map_files/RandomRuins/LavaRuins/lavaland_surface_dead_ratvar.dmm:41:error: unknown type
_maps/map_files/RandomRuins/LavaRuins/lavaland_surface_dead_ratvar.dmm:77:error: undefined type: /obj/structure/lattice/catwalk/clockwork
_maps/map_files/RandomRuins/LavaRuins/lavaland_surface_dead_ratvar.dmm:79:error: unknown type
_maps/map_files/RandomRuins/LavaRuins/lavaland_surface_dead_ratvar.dmm:81:error: undefined type: /obj/structure/lattice/clockwork
_maps/map_files/RandomRuins/LavaRuins/lavaland_surface_dead_ratvar.dmm:83:error: unknown type
_maps/map_files/RandomRuins/LavaRuins/lavaland_surface_dead_ratvar.dmm:97:error: undefined type: /obj/structure/grille/ratvar
_maps/map_files/RandomRuins/LavaRuins/lavaland_surface_dead_ratvar.dmm:103:error: unknown type
_maps/map_files/RandomRuins/LavaRuins/lavaland_surface_dead_ratvar.dmm:106:error: undefined type: /obj/structure/lattice/clockwork
_maps/map_files/RandomRuins/LavaRuins/lavaland_surface_dead_ratvar.dmm:108:error: unknown type
_maps/map_files/RandomRuins/LavaRuins/lavaland_surface_dead_ratvar.dmm:110:error: undefined type: /obj/structure/grille/ratvar/broken
_maps/map_files/RandomRuins/LavaRuins/lavaland_surface_dead_ratvar.dmm:116:error: unknown type
_maps/map_files/RandomRuins/LavaRuins/lavaland_surface_dead_ratvar.dmm:167:error: undefined type: /obj/structure/grille/ratvar
_maps/map_files/RandomRuins/LavaRuins/lavaland_surface_dead_ratvar.dmm:169:error: unknown type
```

**Comment by Sirryan2002** [comment]
> ```
> loading ./map_files/RandomRuins/LavaRuins/lavaland_surface_dead_ratvar.dmm
> _maps/map_files/RandomRuins/LavaRuins/lavaland_surface_dead_ratvar.dmm:39:error: undefined type: /obj/structure/grille/ratvar/broken
> _maps/map_files/RandomRuins/LavaRuins/lavaland_surface_dead_ratvar.dmm:41:error: unknown type
> _maps/map_files/RandomRuins/LavaRuins/lavaland_surface_dead_ratvar.dmm:77:error: undefined type: /obj/structure/lattice/catwalk/clockwork
> _maps/map_files/RandomRuins/LavaRuins/lavaland_surface_dead_ratvar.dmm:79:error: unknown type
> _maps/map_files/RandomRuins/LavaRuins/lavaland_surface_dead_ratvar.dmm:81:error: undefined type: /obj/structure/lattice/clockwork
> _maps/map_files/RandomRuins/LavaRuins/lavaland_surface_dead_ratvar.dmm:83:error: unknown type
> _maps/map_files/RandomRuins/LavaRuins/lavaland_surface_dead_ratvar.dmm:97:error: undefined type: /obj/structure/grille/ratvar
> _maps/map_files/RandomRuins/LavaRuins/lavaland_surface_dead_ratvar.dmm:103:error: unknown type
> _maps/map_files/RandomRuins/LavaRuins/lavaland_surface_dead_ratvar.dmm:106:error: undefined type: /obj/structure/lattice/clockwork
> _maps/map_files/RandomRuins/LavaRuins/lavaland_surface_dead_ratvar.dmm:108:error: unknown type
> _maps/map_files/RandomRuins/LavaRuins/lavaland_surface_dead_ratvar.dmm:110:error: undefined type: /obj/structure/grille/ratvar/broken
> _maps/map_files/RandomRuins/LavaRuins/lavaland_surface_dead_ratvar.dmm:116:error: unknown type
> _maps/map_files/RandomRuins/LavaRuins/lavaland_surface_dead_ratvar.dmm:167:error: undefined type: /obj/structure/grille/ratvar
> _maps/map_files/RandomRuins/LavaRuins/lavaland_surface_dead_ratvar.dmm:169:error: unknown type
> ```

Major bruh moment, time to learn how to map stuff

**Comment by SabreML** [comment]
I don't see a need to remove *any* of the brass structures, really.
All the other clock cult related code can go without much issue, but there's no reason to remove the stuff people actually use for projects sometimes. (Especially since it all looks pretty cool)

**Comment by Sirryan2002** [comment]
> I don't see a need to remove _any_ of the brass structures, really.
> All the other clock cult related code can go without much issue, but there's no reason to remove the stuff people actually use for projects sometimes. (Especially since it all looks pretty cool)

you make a valid point, I'll readd the windows and the grilles but AA wanted to see the lattice go

**Review by Kyep — APPROVED** [review]
By all means remove ratvar_act procs, and code that is useless.
Please don't remove the actual objects, though - aside from looking cool, and being used for events, not having the objects also introduces map issues.

**Comment by Sirryan2002** [comment]
> By all means remove ratvar_act procs, and code that is useless.
> Please don't remove the actual objects, though - aside from looking cool, and being used for events, not having the objects also introduces map issues.

Sounds like a plan, once I get enough freetime I'm gonna go readd those objects I removed and get this PR chugging.

**Comment by Sirryan2002** [comment]
Finally got around to fixing stuff, I've readded all the brass objects and am just sticking with cleansing the code of ratvar_act()

File: code/game/objects/structures/grille.dm [hunk 1]
```diff
@@ -289,9 +282,6 @@
 		color = "#960000"
 		animate(src, color = previouscolor, time = 8)
 
-/obj/structure/grille/ratvar/ratvar_act()
```
Comment (AffectedArc07) [inline code/game/objects/structures/grille.dm:292]: This naming itself just shows how bad it is, the syntax linter cant even handle it

**Review by AffectedArc07 — APPROVED** [review]
Get out you brass bastard.

....brasstard?

File: code/game/objects/structures/grille.dm (hunk 1 above)
Comment (hal9000PR) [reply code/game/objects/structures/grille.dm:292]: What

**Pull request merged**